templates/zerver/unsupported_browser.html

```diff
@@ -0,0 +1,37 @@
+{% extends "zerver/portico.html" %}
+
+{% block portico_content %}
+
+<div class="error_page" style="padding-bottom: 80px;">
```

Comment:
We should just move this control into a CSS class `unsupported_browser_page` rather than using a `style` element.

Reply:
Search for `templates/zerver/app/markdown_help.html` in `tools/linter_lib/custom_check.py`; you'll see we have a few dozen similar rules to cleanup.

Reply:
moved it.

> Search for `templates/zerver/app/markdown_help.html` in `tools/linter_lib/custom_check.py`; you'll see we have a few dozen similar rules to cleanup.

So that's how they are excluded. Thanks for saying.